activesupport/lib/active_support/core_ext/array/grouping.rb

```diff
@@ -113,4 +113,8 @@ def split(value = nil)
       results
     end
   end
+
+  def without(*elements) # :nodoc:
```

Comment:
Should we keep this as `nodoc` or add separate documentation for this from what's in `enumerable.rb`?